example/package.json

```diff
@@ -11,6 +11,7 @@
     "typecheck": "tsc --noEmit",
     "lint": "eslint \"**/*.{js,ts,tsx}\" --fix",
     "lint-ci": "eslint \"**/*.{js,ts,tsx}\" -f @jamesacarr/github-actions",
+    "bundle-install": "cd ios && bundle install",
```

Comment:
Shouldn't this be executed in `example/` instead of `example/ios/`?

Reply:
True, both works, but this is better